foodforthought/src/APICalls/GetRecipeSumary.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from "react";
+// import Ingredients, Intolerance, dietary restriction from food form.js
+
+export default function GetRecipeSummary({  }) {
```

Comment:
name of the file and the function name are different, you are missing an "m" in the name of the file